cmd/kube-proxy/app/server.go

```diff
@@ -792,3 +797,58 @@ func getNodeIP(client clientset.Interface, hostname string) net.IP {
 	}
 	return nodeIP
 }
+
+func reportVersion(client v1core.CoreV1Interface, nodeName string) error {
+	v := version.Get().String()
+	nodeClient := client.Nodes()
+	backoff := wait.Backoff{
```

Comment:
Sure, was just going with the original comment which had me copy other code. I wasn't even sure if we care if this succeeds. Right now its blocking and ignores the error. We could either have it attempt this in the background and ignore the error or attempt and die if it never succeeds.